sky/setup_files/setup.py

```diff
@@ -116,7 +116,8 @@ def parse_readme(readme: str) -> str:
     # Cython 3.0 release breaks PyYAML installed by aws-cli.
     # https://github.com/yaml/pyyaml/issues/601
     # https://github.com/aws/aws-cli/issues/8036
-    'pyyaml<=5.3.1'
+    # <= 3.13 may encounter https://github.com/ultralytics/yolov5/issues/414
+    'pyyaml > 3.13, <= 5.3.1'
```

Comment:
With https://github.com/yaml/pyyaml/pull/702 merged and [PyYAML 6.0.1](https://pypi.org/project/PyYAML/6.0.1/), [awscli 1.29.4](https://github.com/aws/aws-cli/issues/8036#issuecomment-1639136091) released, we might also be able to allow `>=6.0.1` here. Should we test with that too?

Reply:
Generally, people may not constantly upgrade their tools (aws-cli for example) to get those fixes. I think supporting latest versions can be deferred.

Reply:
as we discussed on call, lets defer >=6.0.1 for this PR but let's file an issue and get to it soon